core/src/test/java/com/linecorp/armeria/client/Http2ClientSettingsTest.java

```diff
@@ -149,7 +149,7 @@ public void maxFrameSize() throws Exception {
                     .build();
 
             final HttpClient client = HttpClient.of(clientFactory, "http://127.0.0.1:" + port);
-            final CompletableFuture<AggregatedHttpMessage> future = client.get("/").aggregate();
+            client.get("/").aggregate();
```

Comment:
Hm.. no need to join this future?

Reply:
No, because otherwise it will wait until the response is sent. We send the response below using a Socket, so join() will fail with timeout.